virt/virt-4-13-release-notes.adoc

```diff
@@ -117,6 +114,9 @@ Removed features are not supported in the current release.
 //CNV-19043 Release note: CHANGE Remove rhel6 support
 * Red Hat Enterprise Linux 6 is no longer supported on {VirtProductName}.
 
+//CNV-23499: Carry over/repeat removed feature from version 4.12 
+* Support for the legacy HPP custom resource, and the associated storage class, has been removed. Beginning in {VirtProductName} 4.12, the HPP Operator uses the Kubernetes Container Storage Interface (CSI) driver to configure local storage.
```

Comment:
Beginning in {VirtProductName} 4.12 - > Not sure whether you are stating when this change was made. If so it was made in OpenShift Virtualization 4.10. If you are referring to how things works in the current version (4.13) then change 'Beginning in' to 'In'

Reply:
You're absolutely right, Kevin! Thanks for your comment; I've made the change. Can you take another look? Thanks!